docs/user_guide/how-tos/collab.md

```diff
@@ -14,15 +14,11 @@ If you are using a local installation, your JupyterLab instance is not available
 
    In both cases, you should forward the port of the JupyterLab instance. The default port is `8888`.
 
-2. For a more scalable alternative, consider hosting JupyterGIS on a cloud-based or network-accessible instance. This setup allows multi-user cooperation with authentication and access restriction, without requiring a local installation. Once the instance is created using any of the options below, JupyterGIS needs to be installed on the created instance by opening a terminal window and following [the installation guide](../install.md).
+2. For a more scalable alternative, consider hosting JupyterGIS on a cloud-based or network-accessible instance. This setup allows multi-user cooperation with authentication and access restriction, without requiring a local installation. Once the instance is created using any of the options below, JupyterGIS needs to be installed on the created instance by opening a terminal window and following [the installation guide](/user_guide/install.md).
```

Comment:
Where will users find these cloud-based or network-accessible instances? If they don't know, I imagine this sentence could be rather intimidating.

Reply:
This was another piece of content that I only moved, didn't edit or write originally. Perhaps we could suggest some existing Hubs like CryoCloud or other 2i2c community hubs?